web/angular-wallet/src/app/setup/account/create-active/seed/seed.component.ts

```diff
@@ -43,6 +43,13 @@ export class AccountCreateSeedComponent {
     }
   }
 
+  // speed up for mobile users
+  public handleClick(e) {
```

Comment:
ah, thats the speed up... won't this be less randomized?

Reply:
Yes, but I don't believe enough to create any sort of risk.